File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/bulletcharts/RemoteDataController.java
package com.kendoui.spring.controllers.bulletcharts;

import java.util.List;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.ResponseBody;

import com.kendoui.spring.models.ChartDataRepository;
import com.kendoui.spring.models.AprilSales;

@Controller("dataviz-bullet_charts_remote_data-controller")
@RequestMapping(value="/dataviz/bullet-charts/")
public class RemoteDataController {
    @RequestMapping(value = "/remote-data", method = RequestMethod.GET)
    public String index() {
        return "/dataviz/bullet-charts/remote-data";
    }
    
    @RequestMapping(value = "/remote-data/read", method = RequestMethod.POST)
    public @ResponseBody List<AprilSales> read() {
        return ChartDataRepository.AprilSalesData();
    }
}

File: wrappers/mvc/demos/Kendo.Mvc.Examples/Controllers/DataViz/Bullet_Charts/Remote_DataController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using Kendo.Mvc.Examples.Models;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Bullet_ChartsController : Controller
    {
        public ActionResult Remote_Data()
        {
            return View();
        }

        public ActionResult _AprilSales()
        {
            return Json(ChartDataRepository.AprilSalesData());
        }
    }
}